feature-engineering/categorical_imputer.md

```diff
@@ -0,0 +1,153 @@
+Impute missing categorical values using reference features.
```

Comment:
Grammatically this is a command (telling someone -- the user? -- to do something). Should we say "The Feature Engineering toolkit can impute..." or "Using this toolkit, you can impute..."?